cmd/metal-api/internal/service/switch-service.go

```diff
@@ -176,6 +177,7 @@ func (r switchResource) notifySwitch(request *restful.Request, response *restful
 		s.LastSyncError = sync
 	}
 
+	// FIXME needs retry coverage, not so urgent, will be sent in 5 minutes again
```

Comment:
This part could be "easily" mitigated by writing the switch status into a separate table.

Reply:
https://github.com/metal-stack/metal-api/issues/263